packages/ui-composer/src/MessageComposer/MessageComposerHint.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import { MessageComposerHint } from '.';
+import '@rocket.chat/icons/dist/rocketchat.css';
+
+export default {
+	title: 'Components/MessageComposer/Hint',
+	component: MessageComposerHint,
+} as ComponentMeta<typeof MessageComposerHint>;
+
+export const Editing: ComponentStory<typeof MessageComposerHint> = () => {
+	const helperText = (
+		<>
+			<strong>esc</strong> to cancel · <strong>enter</strong> to save
+		</>
+	);
+
+	return (
+		<MessageComposerHint icon='pencil' helperText={helperText}>
+			Editing message
+		</MessageComposerHint>
+	);
+};
+
+export const ReadOnly: ComponentStory<typeof MessageComposerHint> = () => <MessageComposerHint>This room is read only</MessageComposerHint>;
+
+export const UnencryptedMessage: ComponentStory<typeof MessageComposerHint> = () => (
+	<MessageComposerHint>You're sending an unencrypted message</MessageComposerHint>
+);
```

Comment:
I'm not sure if those variations are actually variations. all of them are the same, only with a different text.

the stories could be "hint, hint with helper text and hint with icon", but I dont see a lot of value adding a story for each text. thats the idea of the component, allow the application set any text inside

Reply:
Yep, make sense, thanks for your view, I'll apply the necessary changes

Reply:
What do you think about the three variations? 

![image](https://github.com/user-attachments/assets/4e3e0ce2-5991-4346-b196-af8a29f10b73)